pages/articles/[slug]/index.js

```diff
@@ -54,10 +54,10 @@ export default function ArticlePage({ projects, theme }) {
 
   return (
     <div>
-      <BackToArticleLink href="/articles">
-        <BackArrowIcon />
+      <StyledLink href="/articles" aria-label="Go back to all articles">
+        <BackArrowIcon title="icon showing an arrow" />
```

Comment:
Hey :) I'm not sure, but doesn't it have to be aria-label here, instead of title?